preamble.tex

```diff
@@ -1,21 +1,15 @@
-\usepackage[margin=1in]{geometry}
-\usepackage{amsmath, amssymb}
-\usepackage{amstext} % for \text macro
-\usepackage{array}   % for new column macro
-\usepackage{commath}
+\usepackage{geometry}
+\usepackage{babel}
```

Comment:
I don't think I need babel, I don't plan on publishing this in different locales